DEVX_ROADMAP.md

```diff
@@ -0,0 +1,46 @@
+# Developer Experience Roadmap
+
+This is a list of the upcoming DevX changes in the next 30 days
+
+## JSON RPC
+*[Gateway deprecation] Sui_syncAccountState will be deprecated and removed.
+* [Gateway deprecation] Gateway’s sui_executeTransaction will be replaced by fullnode’s sui_executeTransaction which has an extra parameter that allows users to choose if a method should block or return immediately.
+* Pagination support for the following APIs.
```

Comment:
are these all APIs or methods of the Sui API?

Reply:
these are a subset that may potential return a very lengthy list and hence requires pagination.